pages/guide.md

```diff
@@ -75,6 +75,8 @@ BOD 18-01 requires federal agencies to set a DMARC policy of `p=reject`, which c
 #### What process should be followed in order to implement email authentication?
 For all second-level domains and all mail-sending hosts generally, make a plan to implement [SPF](/intro#spf--dkim), [DKIM](/intro/#spf--dkim) (mail-senders only), and [DMARC](/intro/#dmarc), **with a goal of setting** `p=reject` **on all second-level domains**.
 
+While implementing email authentication on a second-level domain, it is possible to specify separate policies for its subdomains. However, by October 16th, 2018, the second-level domain must be at `p=reject` without any other policies on subdomains.
```

Comment:
cc: @konklone

This came up on an issue I filed on trustymail: https://github.com/dhs-ncats/trustymail/issues/33. I think what you've written hits at the intent of the directive, but could be further clarified (with many more words). Maybe:

> By default, a `p=` policy applies to the second-level domain **and** to its subdomains unless an `sp` (subdomain policy) tag is used. Setting `p=reject` at the second-level domain is intended by the Directive so as to cascade throughout the zone, protecting all subdomains against spoofing. This is thwarted when a weaker-than-`p=` `sp=` policy is set at the second-level domain. 
> 
> [At one year after BOD issuance](https://cyber.dhs.gov/guide/#checklist), the second-level domain **must be at** `p=reject` **with no** `sp` **policies set at the second-level domain**. 
>
> `p=reject` will not affect mail-sending hosts that use a subdomain, since DMARC-aware mail servers check directly at the (sub)domain in question first (e.g., *_dmarc.sub.domain.gov*) before checking the second-level domain (e.g., *_dmarc.domain.gov*). The second-level domain DMARC policy is applied only when no policy is found directly on the subdomain. See [RFC 7489, section 6.6.3](https://tools.ietf.org/html/rfc7489#section-6.6.3).

Reply:
I stayed away from explicitly mentioning `sp` for a reason: there's a lot of bad advice online recommending its usage, and most people who put `sp=none` in their record never get around to removing it.

The other issue with the text you've suggested is that one can also add `p=none` directly on any subdomain, which leaves the domain open to spoofing and impersonation even with `p=reject` or `sp=reject` on the second-level domain.

Let me think about how to message this succinctly and then I'll update the PR.

Reply:
I appreciate the caution around mentioning `sp` at all, but I'd prefer to exorcise the demon by acknowledging it directly-- otherwise it seems to hang around unbidden, not quite present, not quite not gone. `sp` is in NIST's SP [800-177](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-177.pdf#page=57), after all, and we do want people to read that. 

For federal agencies under the scope of the BOD, DHS scans for and reports on (soon) as much of their namespaces as we have from them/can discover (100k+ .gov domains alone). So, if a subdomain has a `p=none` directly, we'll detect that and flag it, and it will become part of their "compliance denominator".

Reply:
Is this language closer to what you're looking for?:

Subdomains can have their own `p=` policies set, otherwise they inherit the `p=` policy from the second-level domain or it’s `sp=` (subdomain) policy if one exists. Setting `p=reject` at the second-level domain is intended by the Directive so as to cascade throughout the zone, protecting all subdomains against spoofing. This is thwarted when a policy **weaker** than `p=reject` is set on any subdomain either explicitly or through the use of the `sp=` tag.

[At one year after BOD issuance](https://cyber.dhs.gov/guide/#checklist), the second-level domain **must be at `p=reject` with no `sp` policies set at the second-level domain.**

Reply:
Another way to state this might be:

For all second-level domains and all mail-sending hosts generally, make a plan to implement SPF, DKIM (mail-senders only), and DMARC, with a goal of setting `p=reject` on all second-level domains.

Setting `p=reject` at the second-level domain is intended by the Directive so as to cascade throughout the zone, protecting all subdomains against spoofing. A domain that has subbdomains with less restrictive policies is still open to spoofing. There are, however, legitimate reasons for overriding policies on subdomains explicitly while working to properly authenticate services on those subdomains. This can be done by either setting a `p=` policy directly on the subdomain (e.g. on `_dmarc.sub.example.gov`) or by specifying an `sp` policy, or subdomain policy, on the second-level domain.

At one year after BOD issuance, the second-level domain **must be at `p=reject` with no `sp` policies set at the second-level domain or subbdomains with explicit policies less restrictive than `reject`.**